setup.py

```diff
@@ -0,0 +1,42 @@
+"""Setup for kinescope XBlock."""
+
+
+import os
+
+from setuptools import setup
+
+
+def package_data(pkg, roots):
+    """Generic function to find package_data.
+
+    All of the files under each of the `roots` will be declared as package
+    data for package `pkg`.
+
+    """
+    data = []
+    for root in roots:
+        for dirname, _, files in os.walk(os.path.join(pkg, root)):
+            for fname in files:
+                data.append(os.path.relpath(os.path.join(dirname, fname), pkg))
+
+    return {pkg: data}
+
+
+setup(
+    name='kinescope-xblock',
+    version='0.1',
+    description='XBlock that embeds content from Kinescope (https://kinescope.io)',
+    license='AGPL v3',
+    packages=[
+        'kinescope',
+    ],
+    install_requires=[
+        'XBlock',
+    ],
+    entry_points={
+        'xblock.v1': [
+            'kinescope = kinescope:KinescopeXBlock',
+        ]
+    },
+    package_data=package_data("kinescope", ["static", "public"]),
```

Comment:
@kaustavb12, sorry, that was not clear - using `static` (and `js/src`) was the correct way. Please revert these renames and remove the `public` from `setup.py`.

Reply:
@Agrendalath Actually I created the `public` directory and moved the css and js files there because only those locations are accesible through `self.runtime.local_resource_url()` method. This is a [restriction imposed by XBlock](https://github.com/openedx/XBlock/blob/9a634bf1319b9bcf6731e081c703dc2737f60d8b/xblock/core.py#L102-L103) itself.